components/sup/src/manager/spec_watcher.rs

```diff
@@ -29,8 +30,8 @@ use manager::service::ServiceSpec;
 
 static LOGKEY: &'static str = "SW";
 const WATCHER_DELAY_MS: u64 = 2_000;
-const SPEC_FILE_EXT: &'static str = "spec.toml";
```

Comment:
For anyone reading, @fnichol and I had an offline conversation about this. The `/hab` data directory should be completely managed by the `hab` CLI and nothing else. Another configuration management tool, like Chef, should never *ever* write to anything in this directory. Some of the directories here have human readable files including the `/hab/sup` directory, but the files are for the machine. Because of this, we don't need to include extensions on them for our editors. It's best to not leak implementation details in file name in this case. We may be serializing these things out as TOML today, but we could swap to something else in the future if we fancy it!